app/controllers/miq_ae_class_controller.rb

```diff
@@ -1660,6 +1661,20 @@ def x_show
     tree_select
   end
 
+  def refresh_git_domain
+    if params[:button] == "save"
+      git_based_domain_import_service.import(params[:git_repo_id], params[:git_branch_or_tag])
```

Comment:
I don't think so? This is used when the git repository is already initialized, which means they've already put in their credentials. They're just updating to a different branch/tag.